### ISSUE TITLE
Finnish short month names are too long

### ISSUE THREAD

**Issue by GrahamBorland** [ThreeTen/threetenbp#92]
When I format a `Month` in Finnish, with 

```java
month.getDisplayName(TextStyle.SHORT_STANDALONE, Locale.forLanguageTag("fi-FI"))
```

This is the output:

```
tammikuuta
helmikuuta
maaliskuuta
huhtikuuta
toukokuuta
kesäkuuta
heinäkuuta
elokuuta
syyskuuta
lokakuuta
marraskuuta
joulukuuta
```

These names are clearly not abbreviated.

Results are the same for `SHORT`, `SHORT_STANDALONE`,`FULL`, and `FULL_STANDALONE`.

`NARROW` and `NARROW_STANDALONE` give single-letter abbreviations which are a bit _too_ short.

**Comment by GrahamBorland** [comment]
Possibly related: https://github.com/ThreeTen/threetenbp/issues/55

**Comment by GrahamBorland** [comment]
Seen on Android with [ThreeTenABP 1.1.0](https://github.com/JakeWharton/ThreeTenABP) which incorporates threetenbp 1.3.6.

**Comment by GrahamBorland** [comment]
I see correctly abbreviated month names on Android 8.
```
tammik.
helmik.
maalisk.
huhtik.
toukok.
kesäk.
heinäk.
elok.
syysk.
lokak.
marrask.
jouluk.
```

Do they come from the platform, then?

**Comment by sschaap** [comment]
As @jodastephen notes in #55, TTBP resolves localized text (such as month names) through the platform APIs by default. Specifically it uses SPI to resolve [DateTimeTextProvider](https://github.com/ThreeTen/threetenbp/blob/master/src/main/java/org/threeten/bp/format/DateTimeTextProvider.java) which, at runtime, resolves to [SimpleDateTimeTextProvider](https://github.com/ThreeTen/threetenbp/blob/master/src/main/java/org/threeten/bp/format/SimpleDateTimeTextProvider.java) unless otherwise specified. This class provides localized text through **java.text.DateFormatSymbols** which is provided by the platform classpath. You can view the actual codepath by which text is loaded in its [createStore(...)](https://github.com/ThreeTen/threetenbp/blob/142d0017ec443e5e15e811dc558af70ccc3ee84f/src/main/java/org/threeten/bp/format/SimpleDateTimeTextProvider.java#L112) method.

Android includes CLDR-based localizations. You can view the CLDR version in use for each API level on the developer reference page for [java.util.Locale](https://developer.android.com/reference/java/util/Locale) under the "Locale data" heading.

In order to provide other localized text, you could implement your own DateTimeTextProvider and make it SPI loadable. That would allow you to use a different data set, for example based on a more recent CLDR version. However, implementing this is not a trivial task.

**TL;DR** Localized text (such as month names) is provided by Android and not included in TTBP.

**Comment by jodastephen** [comment]
@sschaap has described the problem and solutions nicely!
